spec/abstractops.html

```diff
@@ -266,7 +240,9 @@ <h1>ToTemporalRoundingIncrement ( _normalizedOptions_, _dividend_, _inclusive_ )
         1. Let _maximum_ be _dividend_ − 1.
       1. Else,
         1. Let _maximum_ be 1.
-      1. Let _increment_ be ? GetNumberOption(_normalizedOptions_, *"roundingIncrement"*, 1, _maximum_, 1).
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, « Number », *undefined*, 1).
```

Comment:
```suggestion
      1. Let _increment_ be ℝ(? GetOption(_normalizedOptions_, *"roundingIncrement"*, « Number », *undefined*, 1)).
```
perhaps? this way the comparisons below can be to mathematical values instead of to Numbers

Reply:
Likewise, I'll sort this out in #1413.